datafusion/functions/src/math/nans.rs

```diff
@@ -43,7 +42,10 @@ impl IsNanFunc {
         use DataType::*;
         Self {
             signature: Signature::one_of(
-                vec![Exact(vec![Float32]), Exact(vec![Float64])],
+                vec![
+                    TypeSignature::Exact(vec![Float32]),
+                    TypeSignature::Exact(vec![Float64]),
```

Comment:
Thanks for improving the code style!

i see there is a bunch of codestyle improvement like this. Would it be possible to package them separately so that mechanical changes can be reviewed with less scrutiny and more focus put on important changes?

Reply:
I think it's clear that the change is necessary due to the name conflict with Signature::String, so it would be better to group them together in this case